eng/pipelines/live-test-cleanup-template.yml

```diff
@@ -30,7 +33,8 @@ steps:
         -OpensourceApiApplicationTenant $(opensource-aad-tenant-id) `
         -GithubAliasCachePath ${{ parameters.GithubAliasCachePath }} `
         @subscriptionConfiguration `
-        -Verbose ${{ parameters.AdditionalParameters }}
+        -Verbose ${{ parameters.AdditionalParameters }} `
```

Comment:
For readability I'd put `${{ parameters.AdditionalParameters }}` on its own line.

Reply:
Originally I didn't because I thought if it was empty then we might have errors with the multi-line characters, but testing it I think powershell is flexible enough.